assets/js/app.js

```diff
@@ -146,7 +143,7 @@ class App extends React.Component {
             <main className="container py-5">
               <DismissableAlert message={alertMessage} variant={alertClass} />
               <Switch>
-                <PrivateRoute path="/dashboard" component={Dashboard} authed={user && Object.keys(user).length !== 0 && this.getCurrentUserRole() === "ROLE_REQUESTER"} setAlert={this.setAlert} user={user}/>
+                <PrivateRoute path="/dashboard" component={Dashboard} authed={user && Object.keys(user).length !== 0} setAlert={this.setAlert} user={user}/>
```

Comment:
wird nicht mehr auf das ROLE_REQUESTER abgefragt? bzw würd mich interessieren wie das geprüft wird :D

Reply:
müsste ja eigtl `ROLE_MAKER` sein :D

Reply:
Ich bin davon ausgegangen es gibt 2 Rollen Maker und Requester, alle User haben entweder die Rolle Requester oder die Rolle Maker.
Die Dashbord Komponente funktioniert aktuell sowohl für Maker als auch Requester.
In der Dashbord Komponente wird bei fetchData geprüft ob der User ein Maker oder ein Requester ist. Entsprechend wird dann die eine oder die andere API-Route aufgerufen.

Reply:
macht es ggf. sinn die dashboard views zu splitten?
mit der zeit werden die ja schon eher unterschiedlich sein, oder?

Reply:
Wenn Sie unterschiede haben werde ich Sie spiltten. Bzw. weiterhin eine Dashbord Componente die inkludiert dann eine MakerTable oder eine eine RequesterTable die jeweils eigenständige Componenten sind. Wollte das später angehen im Zuge von https://github.com/print4health/print4health/issues/251 da ändert sich dann ja die View für Maker und Requester.